QUICKSTART.md

```diff
@@ -0,0 +1,287 @@
+# **KCP-Edge** Quickstart
+
+## Required Packages:
+   - docker
+   - kind
```

Comment:
Right now the latest release of kind is v0.18.0.  What release of Kubernetes will that install?  Will that version work with edge-mc?  Which releases of kind install a Kubernetes release that will work with edge-mc?